build/cli.js

```diff
@@ -229,12 +228,26 @@ async function buildDocuments(
       sitemapFilePath,
       zlib.gzipSync(makeSitemapXML(locale, docs))
     );
-    sitemapsBuilt.push(sitemapFilePath);
   }
 
   // do we bother generating the combined sitemaps index file.
   // That means, that if you've done this at least once, consequent runs of
   if (CONTENT_TRANSLATED_ROOT) {
+    // collect all the sitemap.xml.gz files,
```

Comment:
Thanks for catching this. Could we move that as an individual option to the cli.
Like `yarn build --sitemap-index` which just runs this piece and we add it to the build job. While this should always work it can be racy and making it an explicit step helps with that.

Reply:
Hi @fiji-flo, should we preserve the `sitemap.xml` generating here？

Reply:
No, I'd fully move it out of here.